client-react/src/pages/static-app/skupicker/StaticSiteSkuPicker.tsx

```diff
@@ -337,3 +341,21 @@ const SkuCost: React.FC<SkuCostProps> = ({ billingInformation = [] }) => {
 
   return <PricingCalculatorLink />;
 };
+
+const getSWACostString = (billingInformation: CostEstimate[] = [], id: StaticSiteBillingType) => {
+  const { t } = useTranslation();
+
+  if (billingInformation.length > 0) {
+    const meter = billingInformation.find(val => val.id === id);
+    if (meter?.amount) {

```

Comment:
Consider checking only for `undefined` unless it is impossible for the price to be 0.

```suggestion
    if (meter?.amount !== undefined) {
```